pkg/registry/apis/secret/decrypt/v0alpha1/decrypt.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package decrypt.v0alpha1;
```

Comment:
do we want/need this tied to a specific api version?

Reply:
Maybe not, but I wanted to signal that this is experimental and may change, so with your idea of moving it to where we have the k8s spec, it could make sense to evolve it with the spec version.